test/gapic-big_query_storage-v1beta1.ts

```diff
@@ -319,9 +319,7 @@ describe('v1beta1.BigQueryStorageClient', () => {
         projectId: 'bogus',
       });
       // Mock request
-      const request = {
```

Comment:
This is a weird change. We'll need to take a look what's going on here. @xiaozhenliu-gg5 FYI :)